openmetadata-ui/src/main/resources/ui/src/pages/ServiceVersionPage/ServiceVersionMainTabContent.tsx

```diff
@@ -42,15 +43,24 @@ function ServiceVersionMainTabContent({
   serviceDetails,
   entityType,
   changeDescription,
+  isVersionPage = true,
 }: ServiceVersionMainTabContentProps) {
   const { serviceCategory } = useParams<{
     serviceCategory: ServiceTypes;
   }>();
 
   const { fqn: serviceFQN } = useFqn();
+  const { permissions } = usePermissionProvider();
+
+  const editDisplayNamePermission = useMemo(() => {
+    return !isVersionPage
+      ? permissions.databaseService.EditAll ||
+          permissions.databaseService.EditDisplayName
+      : false;
+  }, [permissions]);
 
   const tableColumn: ColumnsType<ServicePageData> = useMemo(
-    () => getServiceMainTabColumns(serviceCategory),
+    () => getServiceMainTabColumns(serviceCategory, editDisplayNamePermission),
```

Comment:
its already version page, why we need to check permission here?

Reply:
Removed edit permission from version page